### PR TITLE
Daily.co app

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Adds the Daily.co app in the App Store.

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [Maybe?] This change requires a documentation update

## How should this be tested?

In the `Apps` pages and in the subsection `Installed`

## Checklist

Hopefully I covered all, open to feedback of course!

### CODE REVIEW

File: packages/app-store/dailyvideo/lib/VideoApiAdapter.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+import prisma from "@calcom/prisma";
 import type { CalendarEvent } from "@calcom/types/CalendarEvent";
 import type { PartialReference } from "@calcom/types/EventManager";
 import type { VideoApiAdapter, VideoCallData } from "@calcom/types/VideoApiAdapter";
 
-import { BASE_URL } from "@lib/config/constants";
-import { handleErrorsJson } from "@lib/errors";
-import prisma from "@lib/prisma";
-
+/** @link hhttps://docs.daily.co/reference/rest-api/rooms/create-room */
```
Comment (zomars) [inline packages/app-store/dailyvideo/lib/VideoApiAdapter.ts:10]:
```suggestion
/** @link https://docs.daily.co/reference/rest-api/rooms/create-room */
```

File: packages/app-store/dailyvideo/lib/VideoApiAdapter.ts [hunk 2]
```diff
@@ -19,8 +19,8 @@ export interface DailyReturnType {
   url: string;
   created_at: string;
   config: {
-    nbf: number;
-    exp: number;
+    nbf: number /* Timestamps expressed in seconds, not in milliseconds */;
+    exp: number /* Timestamps expressed in seconds, not in milliseconds */;
```
Comment (zomars) [inline packages/app-store/dailyvideo/lib/VideoApiAdapter.ts:23]: So these comments actually appear on the IDE.

```suggestion
    /** Timestamps expressed in seconds, not in milliseconds */
    nbf: number;
    /** Timestamps expressed in seconds, not in milliseconds */
    exp: number;
```

imagen

**Review by zomars — APPROVED** [review]
Awesome! Thank you 🙏🏽

File: apps/web/pages/apps/installed.tsx [hunk 3]
```diff
@@ -507,7 +507,7 @@ function ConnectOrDisconnectIntegrationButton(props: {
     );
   }
   /** We don't need to "Connect", just show that it's installed */
-  if (["daily_video", "huddle01_video", "jitsi_video"].includes(props.type)) {
```
Comment (zomars) [inline apps/web/pages/apps/installed.tsx:510]: This caused the Daily.co integration to display a "Connect" button which is the incorrect behaviour. Fixed by setting an `isGlobal` boolean in the metadata.

Comment (leog) [reply apps/web/pages/apps/installed.tsx:510]: Thanks for the update and the heads-up @zomars 👍

**Pull request merged**